sdlib/d/gc/region.d

```diff
@@ -288,24 +326,29 @@ unittest extra_blocks {
 	void* addr1;
 	assert(regionAllocator.acquire(&addr1, 1));
 	assert(addr1 is addr0 + 2 * BlockSize);
+	assert(regionAllocator.nBlocks == 1 + 2);
```

Comment:
It might be better to group asserts about regionAllocator and these about addr.

Reply:
I removed these tests.

Reply:
OK, so these tests are coming back, and I'm rereading this message, and I don't understand the comment.